app/models/facility.rb

```diff
@@ -49,6 +49,13 @@ class Facility < ApplicationRecord
     foreign_key: "assigned_facility_id"
 
   pg_search_scope :search_by_name, against: {name: "A", slug: "B"}, using: {tsearch: {prefix: true, any_word: true}}
+  scope :with_block_region_id, -> {
+    return all unless Flipper.enabled?(:regions_prep)
```

Comment:
I think the feature flag check should go up into the sync code that is building up the query...having it done in the scope itself is hard to understand.  It isn't clear why `return all` is valid if regions_prep is turned off...I think it would make more sense reading in context of the `records_to_sync` method.